sympy/stats/rv.py

```diff
@@ -732,6 +732,10 @@ def probability(condition, given_condition=None, numsamples=None,
     condition = sympify(condition)
     given_condition = sympify(given_condition)
 
+    from sympy import Indexed
+    if isinstance(condition, Indexed):
+        return condition.args[0].probability(condition, given_condition)
```

Comment:
`Indexed` may not have `.probability` as a method.

Reply:
`condition.args[0]` is the `MarkovChain` object, which is having `probability` method. I have created the issue https://github.com/sympy/sympy/issues/16895 for more discussion. Please comment your views so that we can start our work on Markov Chains ASAP.